Clarification on Clock hardware, which is approved at WCA Delegate discretion (#1028)
thewca/wca-regulations: wca-guidelines.md

- 3h2++) [CLARIFICATION] "Stickerless" puzzles that significantly differ from most mass-produced "stickerless" puzzles are only permitted at the discretion of the WCA Delegate.
- 3j+) [REMINDER] In the past, engraved/embossed parts have been permitted. This is no longer permitted.
- 3j++) [CLARIFICATION] On Clock, pins may be distinguishable from any other pin of the same side.

> - 3j1+) [REMINDER] On Clock, loose pins are not considered reasonable wear, at the discretion of the WCA Delegate.

Reviewer: I can't help but feel like this seems to say "loose pins are allowed if the Delegate is fine with it" rather than "loose pins are not allowed, and the Delegate decides if it's not clear". Given that we don't use "at the discretion of the WCA Delegate" for e.g. 3j+++, could we leave that out and lean on 3k instead?
```suggestion
- 3j1+) [REMINDER] On Clock, loose pins (i.e. pins that can toggle or recede using gravity instead of being directly pressed)  are not considered reasonable wear.
```

> - 3j1+) [REMINDER] On Clock, loose pins are not considered a reasonable wear, at the discretion of the WCA Delegate.

Reviewer 2: ```suggestion
- 3j1+) [REMINDER] On Clock, loose pins are not considered reasonable wear, at the discretion of the WCA Delegate.
```

Reviewer 3: Then you shouldn't like the comma in the other 24 places we have one for this phrase in the regulations. I agree that it's not grammatically needed, but it is needed for consistency with the other regulations.

Reviewer 2: Yep, you're right. Initially I checked the regulations to see whether there were any instances of *at the discretion of the WCA delegate* without a comma which did end up existing, but I didn't look much further to see that the ones without commas needed that clause for the sentence to make sense - I'll go update the suggestion.

> - 3j1+) [REMINDER] On Clock, loose pins are not considered a reasonable wear, at the discretion of the WCA Delegate.

Reviewer 3: ```suggestion
- 3j1+) [REMINDER] On Clock, loose pins are not considered reasonable wear, at the discretion of the WCA Delegate.
```

the aritcle is not needed here.

- 3h2++) [CLARIFICATION] "Stickerless" puzzles that significantly differ from most mass-produced "stickerless" puzzles are only permitted at the discretion of the WCA Delegate.
- 3j+) [REMINDER] In the past, engraved/embossed parts have been permitted. This is no longer permitted.
- 3j++) [CLARIFICATION] On Clock, pins may be distinguishable from any other pin of the same side.

> - 3j1+) [REMINDER] On Clock, loose pins (i.e. pins that can toggle or recede using gravity instead of being directly pressed)  are not considered reasonable wear.

Reviewer: Sorry, it looks like I added an extra space here.

Author: We need a new PR if we want to fix it, right?

Reviewer: Yeah.
